pkg/redis/config/definitions.json

```diff
@@ -42,6 +42,98 @@
             "title": "Port",
             "type": "integer"
           },
+          "ssl": {
+            "default": false,
+            "description": "Indicates whether SSL encryption protocol will be used to connect to Redis. It is recommended to use SSL connection if possible.",
+            "instillUIOrder": 4,
+            "title": "SSL Connection",
+            "type": "boolean"
+          },
+          "ssl_mode": {
+            "description": "SSL connection modes. \n  <li><b>verify-full</b> - This is the most secure mode. Always require encryption and verifies the identity of the source database server",
+            "instillUIOrder": 5,
+            "oneOf": [
```

Comment:
I'm not sure if the format can be handled by console auto-form.
@EiffelFly Can you help check again?

Reply:
@xiaofei-du I'll send a commit to update the json-schema to fit our auto-form requirement later.